packages/geo/src/lib/search/search-results/search-results-item.component.ts

```diff
@@ -84,4 +84,27 @@ export class SearchResultsItemComponent {
     });
     moveToOlFeatures(this.map, [olFeature], FeatureMotion.Default);
   }
+
+  /**
+   * On mouse event, mouseenter /mouseleave
+   * @internal
+   */
+  onMouseEvent(event) {
```

Comment:
@aziz-access Have you tried these on mobile..

Reply:
no

Reply:
You have to be sure that this is working on mobile device (or disable the hover on mobile)...

Reply:
okay, i'm working on it